src/migration/1725988464760-SourceMembersIndex.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SourceMembersIndex1725988464760 implements MigrationInterface {
+  name = 'SourceMembersIndex1725988464760';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE INDEX "IDX_source_flags_total_members" ON source USING BTREE (((flags->>'totalMembers')::integer))`,
```

Comment:
@capJavert I updated the index, but I don't seem to see any line that says about the index being utilized:

![Screenshot 2024-09-11 at 5 47 32 PM](https://github.com/user-attachments/assets/6f337836-a2ea-4acc-8b02-7a56fae455a7)

Reply:
Index is ::integer sort is using ::text)::integer

With JSON it comes very precise.
Also make sure you have enough records locally so it actually hits the index 🙏

Reply:
I actually tried that locally too but no luck 😭 
![Screenshot 2024-09-11 at 5 56 50 PM](https://github.com/user-attachments/assets/5bb6b723-a92a-4423-b7d0-95573905a3ca)

Reply:
This was the result. Could it be that the `DESC` part makes it messed up?

![Screenshot 2024-09-11 at 5 58 20 PM](https://github.com/user-attachments/assets/64679075-1c82-4927-b55a-b34bbf81504d)

Reply:
Let me take a look.

Reply:
@sshanzel for me it shows up 
<img width="760" alt="image" src="https://github.com/user-attachments/assets/0402a5a9-6e43-4e51-b230-9e192bd8af6c">

Reply:
did you apply the index migration locally on your db?

Reply:
@capJavert as discussed, the index worked on certain instances, let's see how it behaves in prod data (we will run the explain analyze after merge).